### PR TITLE
JanusGraph 0.3.1 release

### PR DESCRIPTION
**Pull request by pluradj**
Signed-off-by: Jason Plurad <pluradj@us.ibm.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [] Is there an issue associated with this PR? Is it referenced in the commit message?
- [] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [X] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [] Have you written and/or updated unit tests to verify your changes?
- [] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered?
- [] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by pluradj** [comment]
@jerryjch HBase 1.4.4 is already listed as a tested version https://github.com/JanusGraph/janusgraph/blob/master/docs/changelog.adoc

Note that the diff is showing the 0.2.2 updates, which don't include HBase 1.4.4 support.

File: docs/changelog.adoc [hunk 1]
```diff
@@ -52,6 +52,31 @@ For more information on features and bug fixes in 0.3.0, see the GitHub mileston
 
 * https://github.com/JanusGraph/janusgraph/milestone/4?closed=1
 
+=== Version 0.2.2 (Release Date: October 2, 2018)
+
+[source, xml]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.2.2</version>
```
Comment (dpitera) [inline docs/changelog.adoc]: This should be 0.3.1

File: docs/changelog.adoc [hunk 2]
```diff
@@ -52,6 +52,31 @@ For more information on features and bug fixes in 0.3.0, see the GitHub mileston
...
+* Apache HBase 0.98.24-hadoop2, 1.2.6, 1.3.1
+* Google Bigtable 1.0.0
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 1.7.6, 2.4.6, 5.6.2, 6.0.1
+* Apache Lucene 7.0.0
+* Apache Solr 5.5.4, 6.6.1, 7.0.0
+* Apache TinkerPop 3.2.9
+* Java 1.8
+
+For more information on features and bug fixes in 0.2.2, see the GitHub milestone:
```
Comment (dpitera) [inline docs/changelog.adoc]: same here

File: docs/changelog.adoc (hunk 1 above)
Comment (dpitera) [reply docs/changelog.adoc]: NVM I see what is happening here

Comment (pluradj) [reply docs/changelog.adoc]: yeah, the diff is a bit confusing since I pulled in the 0.2.2 release notes

**Pull request merged**